.github/workflows/plugins-test.yaml

```diff
@@ -465,8 +465,16 @@ jobs:
           java-version: 8
       - name: Build SkyWalking Agent
         run: ./mvnw clean package -DskipTests -Pagent >/dev/null
+      - name: Download apm-oracle-10.x-plugin-1.0.1.jar to agent plugins dir.
```

Comment:
we can configure multi-command with `run:.`
I suggest,
``` 
- name: Run oracle 10.2.0.4.0 (1)
   run: |
       curl -L -o ...
       curl -L -o ...
       ./mvnw ...
       ./mvnw ...
       bash ....
```

WDYT?

Reply:
ACK